src/injected/path-snippet-controller.ts

```diff
@@ -36,19 +36,11 @@ export class PathSnippetController {
 
         this.elementFinderByPath.processRequest(message).then(
             result => {
-                this.sendBackElementFromPath(result, path);
+                this.addCorrespondingSnippet(false, result);
             },
             err => {
-                this.sendBackElementFromPath('error', path);
+                this.addCorrespondingSnippet(true, 'No code snippet is mapped to: ' + path);
```

Comment:
Discussed offline, this is much better than earlier when we had three instances of 'no code snippet...'. We can still improve it further by moving the UX string to the JSX layer in an upcoming PR.